src/Components/Home.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react'
+
+
+function Home(){
+  return(
+    <div>
```

Comment:
What is this `h1` for?

Reply:
we were told to create a dummy route for the landing page